javascript/javascript3/week1/giphyApp/giphyApp.js

```diff
@@ -0,0 +1,45 @@
+const btnGetGif = document.getElementById('btnGetGif');
+const numberInput = document.getElementById('numberInput');
+const displayResult = document.getElementById("searchResult");
+
+btnGetGif.addEventListener("click", getDataFromGify);
+numberInput.addEventListener("input", limitNumberOfGifs);
+
+let imgUrlArray;
+
+function getDataFromGify() {
+    const searchInput = document.getElementById("searchInput").value;
+    if (searchInput) {
+        displayResult.innerHTML = "Processing"
+        const url = `https://api.giphy.com/v1/gifs/search?api_key=TiUGh7M7GYEDMAHpGFi43098D7Ort4ij&q=${searchInput}&lang=en`
+        fetchGifs(url);
+    } else {
+        displayResult.innerHTML = "Specify search word first"
+    }
+}
+
+function limitNumberOfGifs() {
+    const number = inputNumber.value;
```

Comment:
The name of the variable should `numberInput` in the following line `const number = numberInput.value`